ui/src/test/java/edu/wpi/grip/ui/codegeneration/WatershedGenerationTest.java

```diff
@@ -85,6 +86,7 @@ void generatePipeline() {
     }
   }
 
+  @Ignore("Grip Watershed is broken so it doesn't make sense to fix the generated watershed.")
```

Comment:
Yay! We were debating fixing the GRIP Watershed Operation. According to @osulacam currently what it doesn't isn't what a watershed actually is supposed to do. Glad to hear that it is being done.